llama_index/node_parser/interface.py

```diff
@@ -62,7 +62,10 @@ def get_nodes_from_documents(
 
             if self.include_metadata:
                 for node in nodes:
-                    if node.ref_doc_id is not None:
+                    if (
```

Comment:
Yea I'm really confused how a ref doc id could not be in the dictionary 

Unless the underlying parser is creating nodes that point to other nodes?

Reply:
I think the root issue is here

https://github.com/run-llama/llama_index/blob/6c33ae7f780fda4a54642b9d201a66e3c1c7e48b/llama_index/node_parser/relational/unstructured_element.py#L143